acceptance/tests/windows/eventlog.rb

```diff
@@ -12,16 +12,16 @@
   # get remote time
   now = on(agent, "#{ruby_command(agent)} -e \"puts Time.now.utc.strftime('%m/%d/%Y %H:%M:%S')\"").stdout.chomp
 
-  # it should fail to start since parent directories don't exist
-  confdir = "/does/not/exist"
-
   # generate an error, no master on windows boxes
-  on agent, puppet_agent('--server', '127.0.0.1', '--test', '--confdir', confdir), :acceptable_exit_codes => [1]
+  # we use `agent` because it creates an eventlog log destination by default,
+  # whereas `apply` does not.
+  on agent, puppet('agent', '--server', '127.0.0.1', '--test'), :acceptable_exit_codes => [1]
 
   # make sure there's a Puppet error message in the log
   # cygwin + ssh + wmic hangs trying to read stdin, so echo '' |
   on agent, "cmd /c echo '' | wmic ntevent where \"LogFile='Application' and SourceName='Puppet' and TimeWritten >= '#{now}'\"  get Message,Type /format:csv" do
-    fail_test "Event not found in Application event log" unless
-      stdout =~ /Cannot create [a-z]:\/does\/not\/exist.*,Error/mi
+    if stdout !~ /Could not retrieve catalog.*skipping run,Error/mi
+      fail_test "Event not found in Application event log"
+    end
   end
```

Comment:
do we prefer the stdout !~ //  pattern over the previous fail.. unless?

also, we should probably be specifying the result.stdout to use, otherwise i'm not sure what stdout this is...
``` on agent, "blah" do |result|
  assert_match('blah 2', result.stdout, 'omg')
end
```